### PR TITLE
Headline test on /new (Fixes #6851)

### LINKED ISSUE #6851: exp01 - Headline Test on /new: Replace the headline “The new Firefox”
## Description

We know from past experience that a headline change can increase conversion rate on the /new page and that action-driven headlines that offer a benefit drive clicks. This experiment will test the hypothesis that if we replace the old headline and sub-head on the /new download page with benefits driven copy, we’ll increase the session-to-download install rate leading to an increase in active days generated by downloads because our user segmentation research demonstrates a large portion of our user base is interested in how the features of the product can improve their browsing experience.

[EN locales - Copy](https://docs.google.com/document/d/1NDXgfsCDMZpGSYOF12QO5-tD7OWVUU7GNOqNoNGkSgY/edit?usp=sharing)
[DE locale - Copy]
[Manapage for experiment parameters](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=90738380)

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | 2 days|
| PM | 1 day |
| ENG | 3 |
| SEO | 0 |
| ANA | ? |
| DES | 0 |
| COPY | 0 |

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Experiment is launched and live across 5 locales in Sprint 3 (Mar 4-22)

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [x] Getting  'side-step' of stub attribution set up
- [ ] Not having the FR localization - BUT we can and should launch without it if it's not delivered in time

#### Tasks
- [ ] Code content for each of 4 variation in copy across each of the 5 locales (20 vars total)
- [x] Code Traffic Cop according to spec set up in [manapage](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=90738380)
- [x] set up 'side-step' of stub attribution - dependency on issue TBD  - 2 days to make this reusable

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a multi-variant headline test for English and German locales on `/firefox/new/`.
- Removes code for prior "how to install" experiment.

## Issue / Bugzilla link
#6851

## Testing
Demo URLs:

- https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/new/
- https://bedrock-demo-agibson.oregon-b.moz.works/en-CA/firefox/new/
- https://bedrock-demo-agibson.oregon-b.moz.works/en-GB/firefox/new/
- https://bedrock-demo-agibson.oregon-b.moz.works/de/firefox/new/

Experiment config: https://github.com/mozilla/bedrock/issues/6851#issuecomment-472804913

- [ ] Correct headline variations are shown for each locale.
- [ ] Each locale uses correct cohort sizing.
- [ ] How to install experiment code is removed.

### CODE REVIEW

File: docs/stub-attribution.rst [hunk 1]
```diff
@@ -72,8 +72,10 @@ the page template where an experiment exists:
 .. code-block:: html
 
     {% block stub_attribution %}
+      <!--[if !lte IE 8]><!-->
```
Comment (stephaniehobson) [inline docs/stub-attribution.rst]: Is it worth putting the conditional comments around the the outside of the sub_attribution block so they can't be missed?

Comment (stephaniehobson) [reply docs/stub-attribution.rst]: `{% if self.stub_attribution() %} <!--[if !lte IE 8]><!--> {% endif %}`

Comment (alexgibson) [reply docs/stub-attribution.rst]: Not quite sure I follow this suggestion, but we don't want to exclude old IE's from regular stub attribution, only our experiment code.

**Comment by alexgibson** [comment]
This experiment is now complete and ready for code review.

**Comment by alexgibson** [comment]
The en-US locale has now been removed from this experiment so that it does not conflict with https://github.com/mozilla/bedrock/issues/6899

**Comment by alexgibson** [comment]
> This looks good to me, though I'm not sure how best to verify the stub attribution.

We have unit tests for the stub attribution code, so I don't think we need to manually verify that it works each time we use it in an experiment, only that it is called correctly.

**Comment by alexgibson** [comment]
> I think we should hold off merging until we have a plan and can make any needed adjustments. So a tentative r+ for the moment.

So I'm not sure if there's anything more to do here for now. This experiment is removed from en-US as it stands, but we need to try and get both of these PRs into master in order to make them both play together. I think we should probably merge this?

**Pull request merged**